README.md

```diff
@@ -1,48 +1,21 @@
 # Introduction
 
-The connection between Mews \(as a PMS\) and a channel manager is a 2-way connection.   
-The following document defines both the Mews \(as PMS\) side, and also defines the API on the channel manager side.   
-The Mews side mainly accepts bookings from the channel manager, while the channel manager side accepts inventory updates from the PMS and then distributes them to connected channels.
+Welcome to the __Mews Channel Manager API__. This is the Mews API for channels and distribution, supporting two main use cases:
```

Comment:
The word 'channels' is used here deliberately, instead of Channel Managers. But perhaps it could be better worded like this - "Welcome to the __Mews Channel Manager API__. This is the Mews API for distribution and sales channels, supporting two main use cases:"